### PR TITLE
Bug 944186-zone-css-de-dup: Add separate zone style sheets, do not remove combined zones.css yet

### PR DESCRIPTION
**Pull request by stephaniehobson**
Adds separate CSS files for individual zones, to be pulled in using the admin interface after PR #4209.

Does not remove the individual zone styles from zones.scss yet, so there's a bunch of duplicated styles. A follow up PR will remove that duplication once we've transitioned to using the new separate files.

Removed `.articulated-diagram-component`  after consulting with Will.

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Not possible to test without #4209 but it is theoretically possible to merge this now without messing up the styles on anything. @jwhitlock do you want to give it a look on an instance with the new data model?

Zones missing their own zone-* files use the default, except Persona, it gets the archive style sheet.

File: kuma/static/styles/zones/components/articulated-diagram.scss [hunk 1]
```diff
@@ -1,10 +0,0 @@
-.articulated-diagram-component {
```
Comment (jwhitlock) [inline kuma/static/styles/zones/components/articulated-diagram.scss]: You'll need to remove the ``@import`` of this file in ``kuma/static/styles/zones.scss``.

File: kuma/static/styles/zones/components/dev-program.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
-The Connect zone used to be called the Dev Program.
-These classes style that page
+Styles for connect zone
+- formerly the dev program page hence the class names
```
Comment (jwhitlock) [inline kuma/static/styles/zones/components/dev-program.scss]: 👍 comment updates

File: kuma/static/styles/zones/connect.scss [hunk 3]
```diff
@@ -8,7 +8,7 @@ body[data-slug^='Mozilla/Connect'] {
     }
 
     .masthead-text,
-    .zone-landing h1 {
+    &.zone-landing h1 {
```
Comment (jwhitlock) [inline kuma/static/styles/zones/connect.scss]: Curious - does this fix something?

File: kuma/static/styles/zone-ten.scss [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+@import 'zones/components/promo10';
+@import 'zones/components/quote';
+
+
+/*
+header modifications
+====================================================================== */
+.zone {
+    background-color: #0952a8;
+    background-image: url($path-to-images + 'zones/mdn10/head.png'), url($path-to-images + 'zones/blueprint.png'), linear-gradient(to bottom, #153985, #044ca2 600px);
```
Comment (jwhitlock) [inline kuma/static/styles/zone-ten.scss]: No file ``kuma/static/img/zones/blueprint.png``

File: kuma/static/styles/zone-ten.scss [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    background-position: center -37px;
+}
+
+.zone.zone-landing {
+    background-position: center -6px;
+}
+
+@media #{$media-query-mobile-only} {
+    .zone {
+        background-image: url($path-to-images + 'zones/blueprint.png'), linear-gradient(to bottom, #153985, #044ca2 600px);
```
Comment (jwhitlock) [inline kuma/static/styles/zone-ten.scss]: No file ``kuma/static/img/zones/blueprint.png``.  Are there other changes needed from previous PRs?

File: kuma/static/styles/zone-addons.scss [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+====================================================================== */
+@import 'zones/components/webext-summary-compat-table';
+
+
+/* =================================================================== */
+
+.addon-sdk-api-name {
+    background-color: rgba(212, 221, 228, .15);
+    padding: 3px;
+}
```
Comment (stephaniehobson) [inline kuma/static/styles/zone-addons.scss]: To delete them in a follow up PR once we've switched their zones to include the individual zone style sheets.

File: kuma/static/styles/zones/connect.scss (hunk 3 above)
Comment (stephaniehobson) [inline kuma/static/styles/zones/connect.scss]: Yeah, the text colour difference you identified in the other PR.

File: kuma/static/styles/zone-ten.scss (hunk 4 above)
Comment (stephaniehobson) [inline kuma/static/styles/zone-ten.scss]: Goddammit git.

Comment (stephaniehobson) [inline kuma/static/styles/zone-ten.scss]: 👍

**Pull request merged**